### ISSUE TITLE
Save and Reboot button missing on Windows Tablet

### ISSUE THREAD

**Issue by pilotgary** [cleanflight/cleanflight-configurator#436]
I have a Chuwi 8" Hi8Pro Tablet for use at the flying field.  When I try to configure my board I can connect and make changes, but there is no "Save and Reboot" button at the bottom of the screen to make the changes permanent.  I have tried the tablet in desktop and tablet mode and with Chrome windowed and maximized.  Windows 10 on the tablet is up to date as is Chrome.

Is there an option to have the save and reboot button moved into the header instead of the footer?

**Comment by Austinthemighty** [comment]
can you provide a screen shot

**Comment by pilotgary** [comment]
I've got the tablet with me today, but not a flight controller.  I'll get some screen shots to you as soon as I can.

**Comment by pilotgary** [comment]
Here are some screen grab.  The problem is occurring when Chrome is maximised. When I turned maximise off the window was bigger than the screen with the save and reboot button partially visible in landscape, but I couldn't resize the window to see it all as the handle was off screen.
When I reduced the screen resolution the window resized and I could see the whole button.  Unfortunately, when I returned the screen resolution to native the window the was off the screen again and maximising the app lost the save an reboot button.

All screen grabs are fullscreen.

![landscape1920x1200fullscreen](https://cloud.githubusercontent.com/assets/7470508/21177423/13b510da-c1e3-11e6-9a2a-27b89f79e40a.PNG)
![portrait1920x1200fullscreen](https://cloud.githubusercontent.com/assets/7470508/21177424/13f84742-c1e3-11e6-8258-f19c6c908bd6.PNG)
![landscape1920x1200window](https://cloud.githubusercontent.com/assets/7470508/21177422/13b08f92-c1e3-11e6-8dc2-5cc1745d66fa.PNG)
![landscape1280x800window](https://cloud.githubusercontent.com/assets/7470508/21177421/13ad74ce-c1e3-11e6-9734-2ea9efc69a84.PNG)

**Comment by Austinthemighty** [comment]
that is odd i will look into i, what is your screens resolution

**Comment by skaman82** [comment]
Just tested this on my 8" Windows tablet just now and the interface works as intended. Sure, the portrait orientation is broken, but it is impossible to fit everything into that tight space - it is not a mobile app after all.
![screenshot 1](https://cloud.githubusercontent.com/assets/11768024/21617060/9f4471d4-d1e4-11e6-8dab-1d845091c958.png)

**Comment by pilotgary** [comment]
I think I've found the problem.  While checking the screen resolution (1280x800) I noticed in the display settings that the "... size of text, apps and other items" was set to 200%.  
![landscape1280x800window](https://cloud.githubusercontent.com/assets/7470508/21720134/64323716-d419-11e6-9876-a0bf0775d0ba.PNG)
I reduced this to 100% and the "Save and Reboot" button appeared with the app maximised".  I increased the scale back to 150% and then 175% and the button remained.  
![displaysettings](https://cloud.githubusercontent.com/assets/7470508/21720051/e3c4db24-d418-11e6-90f1-37ff38a25ce6.PNG)
Taking it back to 200% caused it to vanish again.
Perhaps skaman82 could test if this behaviour is common to his tablet or intermittent across models.
Although small, 175% is a useable size, especially using a pointer rather than a finger. 
With this workaround, I'm happy for you to close the ticket or mark it as low priority.
Thank you for your time and great software.
